moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -271,12 +264,11 @@ ServoParameters ServoParameters::get(const std::string& ns,
 
   // ROS Parameters
   parameters.use_gazebo = node_parameters->get_parameter(ns + ".use_gazebo").as_bool();
-  parameters.status_topic = node_parameters->get_parameter(ns + ".status_topic").as_string();
+  parameters.status_topic = "/servo_server/status";
```

Comment:
Can you make these constants in the anonymous namespace above like Logger and then assign the constant to the parameters member? Something like

```
namespace {
const std::string SERVO_STATUS_TOPIC = "/servo_server/status";
}
...
parameters.status_topic = SERVO_STATUS_TOPIC;
```